### PR TITLE
Backup Configuration for stone-prod-p01 and stone-stage-p01

### PR DESCRIPTION
**Pull request by manish-jangra**

### CODE REVIEW

File: components/backup/base/privatelink/backup-tenants-schedule.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+apiVersion: velero.io/v1
```
Comment (hugares) [inline components/backup/base/privatelink/backup-tenants-schedule.yaml:1]: This one looks very similar to the schedule we use on public clusters, can we reuse the same one?

File: components/backup/base/privatelink/backup-toolchain-host-schedule.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+apiVersion: velero.io/v1
```
Comment (hugares) [inline components/backup/base/privatelink/backup-toolchain-host-schedule.yaml:1]: This one looks very similar to the schedule we use on public clusters, can we reuse the same one?

File: components/backup/base/privatelink/backup-toolchain-member-schedule.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+apiVersion: velero.io/v1
```
Comment (hugares) [inline components/backup/base/privatelink/backup-toolchain-member-schedule.yaml:1]: This one looks very similar to the schedule we use on public clusters, can we reuse the same one?

File: components/backup/base/privatelink/backup-tenants-schedule.yaml (hunk 1 above)
Comment (hugares) [reply components/backup/base/privatelink/backup-tenants-schedule.yaml:1]: I checked and they are exactly the same. I have a proposal to remove duplication, let me open another PR with one commit on top of yours for you to see. If you agree with the change, you can apply the changes in this PR.

Comment (hugares) [reply components/backup/base/privatelink/backup-tenants-schedule.yaml:1]: here it is: https://github.com/redhat-appstudio/infra-deployments/pull/3315

Comment (manish-jangra) [reply components/backup/base/privatelink/backup-tenants-schedule.yaml]: thanks @hugares, added what you suggested.

**Comment by hugares** [comment]
@manish-jangra I cannot believe I missed this at review... The backup ArgoCD application is not deployed on downstream clusters. We need another PR to deploy backup application on downstream-production.

**Comment by manish-jangra** [comment]
> @manish-jangra I cannot believe I missed this at review... The backup ArgoCD application is not deployed on downstream clusters. We need another PR to deploy backup application on downstream-production.

Sorry, I missed it too. I see that backup application is added into `delete-applications.yaml` with `$patch: delete`

**Pull request merged**